Update/correct URL patterns for Cerner Central metadata and sso endpo…

srbach opened the pull request:
…ints

The only pattern that Cernercentral.com recognizes at this time to facilitate SSO through SAML is through the use of a subdomain or "enterprise" identifier as we call them. Like `https://<instancename>.cernercentral.com/session-api/protocol/saml2/sso` where <instancename> is the subdomain or enterprise.

azurecla commented:
Hi __@maxluk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (maxluk). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

srbach committed:
Update/correct URL patterns for Cerner Central metadata and sso endpoints

The only pattern that Cernercentral.com recognizes at this time to facilitate SSO through SAML is through the use of a subdomain or "enterprise" identifier as we call them. Like `https://<instancename>.cernercentral.com/session-api/protocol/saml2/sso` where <instancename> is the subdomain or enterprise.
